src/services/administration/additional-settings/provider/provider-add/ProviderAddPage.vue

```diff
@@ -7,26 +7,106 @@
             @goBack="$router.go(-1)"
         />
         <p-divider />
-        <p>Add Page</p>
+        <p-data-loader class="flex-grow" :data="providerList" :loading="loading">
+            <div class="provider-item-wrapper">
+                <provider-list-card v-for="item in providerList" :key="item.name" :provider-item="item"
+                                    :visible.sync="visibleInstallModal"
+                                    :selected-provider.sync="selectedProvider"
+                                    @select="handleSelectProvider"
+                />
+            </div>
+        </p-data-loader>
+        <!--song-lang-->
+        <p-button-modal
+            :header-title="$t('Would you like to install', {provider: selectedProvider})"
+            :visible.sync="visibleInstallModal"
+            size="sm"
+            @confirm="handleConfirmInstallProvider"
+        />
+        <!--song-lang-->
+        <p-icon-modal
+            :visible.sync="visibleResultModal"
+            icon-name="ic_done"
+            :header-title="$t('has been successfully added.', {provider: selectedProvider})"
+            :button-text="$t('Done')"
+            size="md"
+        />
     </section>
 </template>
 
 <script lang="ts">
 
 
-import { PPageTitle, PDivider } from '@spaceone/design-system';
+import {
+    computed, reactive, toRefs,
+} from '@vue/composition-api';
 
+import {
+    PPageTitle, PDivider, PDataLoader, PButtonModal, PIconModal,
+} from '@spaceone/design-system';
+
+import { store } from '@/store';
+
+import ProviderListCard from '@/services/administration/additional-settings/provider/provider-add/modules/ProviderListCard.vue';
 
 export default {
     name: 'ProviderAddPage',
     components: {
         PPageTitle,
         PDivider,
+        PDataLoader,
+        ProviderListCard,
+        PButtonModal,
+        PIconModal,
     },
     setup() {
-        return {
+        const state = reactive({
+            providerList: computed(() => Object.keys(store.state.reference.provider.items).map(k => ({
+                name: k,
+                icon: store.state.reference.provider.items[k].icon,
+                label: k,
+                color: store.state.reference.provider.items[k].color,
+                installed: false,
+            }))),
```

Comment:
installed를 제외하면 store.state.reference.provider.items와 타입이 똑같아 보입니다! 이게 어떨까유
```suggestion
            providers: computed(() => store.state.reference.provider.items),
            providerList: computed(() => state.providers.map((d) => {
                ...d,
                installed: false,
            }))),
```

Reply:
`store.state.reference.provider.items`가 배열이 아니라 객체더라구요!